package.json

```diff
@@ -37,13 +37,15 @@
     "signal-github": "./scripts/signal-deploy-to-pr.sh"
   },
   "dependencies": {
+    "ember-auto-import": "^1.12.0",
     "ember-cli-babel": "^7.26.6",
     "ember-cli-htmlbars": "^5.6.2",
     "ember-cli-sass": "^10.0.1",
     "ember-cli-string-utils": "^1.1.0",
     "ember-click-outside": "^2.0.0",
     "ember-prop-modifier": "^1.0.1",
-    "ember-truth-helpers": "^3.0.0"
+    "ember-truth-helpers": "^3.0.0",
+    "lodash": "^4.17.21"
```

Comment:
**question** : pourquoi ça ça a changé ?

Reply:
on a pas accès au devDependencies dans l'addon et je devais utiliser loash et pour importer lodash il fallait embre auto import

Reply:
Si on veut éviter ca, on peut utiliser du natif en remplaçant le `uniq(map(list))` par un `new Set(list.map)`

Reply:
Il y a des raisons pour vouloir éviter d'utiliser lodash dans l'addon ?

Reply:
Je favoriserais le natif tant que possible, mais pas d'objection à ajouter lodash

Reply:
Bon pas ouf le Set parce que un set ne gère pas la notion d'ordre

Reply:
On fait un sort juste après donc ce n'est pas génant dans ce cas précis

Reply:
ouais, mais il faut transformer le set en array pour faire un sort et je sais pas encore pourquoi mais ça m'embête un peu.

Je trouve pas que l'utilisation du Set pour garder les éléments unique rende cette intention hyper évidente dans le code (bien que je trouve l'idée élégante). Du coup j'ai viré lodash et j'ai viré les doublons à la main.

Reply:
C'est pourtant un "classique". 
https://miguel-albrecht.medium.com/performance-of-javascript-array-ops-2690aed47a50